website/static/js/components/profilePagination.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+var m = require('mithril'); // exposes mithril methods, useful for redraw etc.
+var $osf = require('js/osfHelpers');
+var iconmap = require('js/iconmap');
+var lodashFind = require('lodash.find');
+var mHelpers = require('js/mithrilHelpers');
+var Raven = require('raven-js');
+
+
+var MAX_PAGES_ON_PAGINATOR = 7;
+var MAX_PAGES_ON_PAGINATOR_SIDE = 5;
+
+var ProfilePagination = function(ctrl, buildUrlFunction) {
```

Comment:
Is there actually anything profile-specific about this code? If not, then this file and function should be renamed.